pyqtgraph/graphicsItems/ViewBox/tests/test_ViewBox.py

```diff
@@ -31,7 +31,8 @@ def init_viewbox():
     
     g = pg.GridItem()
     vb.addItem(g)
-    
+    desktop = app.desktop().screenGeometry()
+    print("\n\nDesktop Resolution: {} x {}\n\n".format(desktop.width(), desktop.height()))
```

Comment:
It actually seems like this is useful enough to go into `pg.systemInfo()`, in which case it would be printed whenever we run `pyqtgraph/test.py`

Reply:
I considered doing this in `pg.systemInfo()` but it doesn't have a `QApplication` running at that point from what I could tell.  Should I create a `QApplication` instance?

Reply:
Migrated this to `test.py` per offline discussion in 4b26519